core/src/main/java/org/apache/seata/core/rpc/netty/http/HttpDispatchHandler.java

```diff
@@ -82,13 +56,13 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpRequest httpRequest)
             HttpInvocation httpInvocation = ControllerManager.getHttpInvocation(path);
 
             if (httpInvocation == null) {
-                sendErrorResponse(ctx, HttpResponseStatus.NOT_FOUND, keepAlive);
+                sendErrorResponse(ctx, HttpResponseStatus.NOT_FOUND, false);
                 return;
             }
 
-            HttpContext httpContext = new HttpContext(httpRequest, ctx, keepAlive);
+            HttpContext<HttpRequest> httpContext = new HttpContext<>(httpRequest, ctx, keepAlive,HttpContext.HTTP_1_1);
             ObjectNode requestDataNode = OBJECT_MAPPER.createObjectNode();
-            requestDataNode.putIfAbsent("param", ParameterParser.convertParamMap(queryStringDecoder.parameters()));
+            requestDataNode.put("param", ParameterParser.convertParamMap(queryStringDecoder.parameters()));
```

Comment:
The origin method in here is putIfAbsent

Reply:
> The origin method in here is putIfAbsent

There is no need to use the putIfAbsent method, so I will use the set method instead.